React/Modules/RCTAnimationDriverNode.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+#import "RCTBridgeModule.h"
+
+@class RCTValueAnimatedNode;
+@interface RCTAnimationDriverNode : NSObject
+
+@property (nonatomic, readonly) NSNumber *animationId;
+@property (nonatomic, readonly) NSNumber *outputValue;
+
+@property (nonatomic, readonly) BOOL animationHasBegun;
+@property (nonatomic, readonly) BOOL animationHasFinished;
+
+- (instancetype)initWithId:(nonnull NSNumber *)animationId
```

Comment:
@nicklockwood I busted id for animation to match what is happening on the JS/Android side. For some reason they use ID only for the animation tag.